doc/modules/clustering.rst

```diff
@@ -137,8 +137,13 @@ mixture models. KMeans can be seen as a special case of Gaussian mixture
 model with equal covariance per component.
 
 :term:`Transductive <transductive>` clustering methods (in contrast to
-:term:`inductive` clustering methods) are not designed to be applied to new,
-unseen data.
+:term:`inductive` clustering methods) are not
+designed to be applied to new, unseen data.
```

Comment:
Hi @virchan thanks for the suggestions.
I've seen yesterday that when I launched the updated PR, it started directly from the own repository instead from my fork repository.
I received the notifications about issues on the CI failed tests, and for instance I report the comment from the linter step:
""
Error: Unable to download artifact(s): Artifact not found for name: lint-log
        Please ensure that your artifact is not expired and the artifact was uploaded using a compatible version of toolkit/upload-artifact.
""
Maybe could come from not updated files in the own repository?
To avoid troubles, I close the pull request, I'll update my fork and local repository, and then I'll open a new one.